foodalert/static/foodalert/js/components/form-template.vue

```diff
@@ -65,6 +65,11 @@
                     state-value="needContainer"
```

Comment:
Remove this labelled input so that there aren't duplicate inputs for bringing containers.